app/models/health_care_application.rb

```diff
@@ -205,7 +211,7 @@ def form_submission_id
   end
 
   def parsed_form
-    @parsed_form ||= JSON.parse(form)
+    @parsed_form ||= form.present? ? JSON.parse(form) : nil
```

Comment:
Added this logic to validate that we have a form before trying to parse it. When I updated `log_submission_failure_details` to validate if present it broke the SubmissionJob and AnonSubmissionJob specs. Validating the form here fixes them. I also added specs in the model specs for this method.